site/community/bookkeeper_proposals.md

```diff
@@ -85,7 +85,7 @@ using Google Doc.
 
 This section lists all the _bookkeeper proposals_ made to BookKeeper.
 
-*Next Proposal Number: 41*
+*Next Proposal Number: 42*
```

Comment:
@eolivelli Since 42 is already accepted, shouldn't this be 43?

Reply:
you are right.
I have fixed it